docs/BREAKINGCHANGES.md

```diff
@@ -0,0 +1,217 @@
+
+
+### Rename $IsOSX to $IsMacOS [#4700](https://github.com/PowerShell/PowerShell/issues/4700)
+The naming in PowerShell should be consistent with our naming andd conform to Apple's use of macOS instead of OSX, however, for readability and consistently we are staying with Pascal casing.
+
+
+### Prepare for BOM-less UTF-8 default character encoding with respect to $OutputEncoding and console code page  [#4681](https://github.com/PowerShell/PowerShell/issues/4681)
+Change `$OutputEncoding` to be utf8 without BOM rather than ASCII as BOM-less UTF-8 character encoding is the default for PowerShell Core.
+
+
+### Make error message consistent when invalid script is passed to -File, better error when passed ambiguous arg [#4573](https://github.com/PowerShell/PowerShell/issues/4573)
+Change the exit codes of pwsh.exe to align with Unix conventions
+
+
+### Removal of `LocalAccount` and cmdlets from  `Diagnostics` modules. [#4302](https://github.com/PowerShell/PowerShell/issues/4302) [#4303](https://github.com/PowerShell/PowerShell/issues/4303)
+Due to unsupported APIs, the `LocalAccounts` module and the `Counter` cmdlets in the `Diagnostics` module were removed until a better solution is found. 
+
+
+### Remove UTC and SQM telemetry code [#4190](https://github.com/PowerShell/PowerShell/issues/4190)
+TODO: Should this still be in the doc if the change is not visible to the user and is about telemetry collection?
+
+
+### Executing powershell script with bool parameter doesnt work [#4036](https://github.com/PowerShell/PowerShell/issues/4036)
+Previously, using powershell.exe (now pwsh.exe) to execute a PowerShell script using `-File` provided no way to pass $true/$false as parameter values.  Support for $true/$false as parsed values to parameters was added. Switch values are also supported as currently documented syntax doesn't work.
+
+
+### Remove ClrVersion property from $PSVersionTable [#4027](https://github.com/PowerShell/PowerShell/issues/4027)
+The ClrVersion property of `$PSVersionTable` is not useful with CoreCLR,  end users should not be using that value to determine compatibility.
+
+
+### Change positional parameter for powershell.exe from -Command to -File [#4019](https://github.com/PowerShell/PowerShell/issues/4019)
+Enable shebang use of PowerShell on non-Windows platforms. This means on Unix based systems,
+you can make a script executable which would invoke PowerShell automatically rather than explicitly invoking pwsh.
+This also means that you can now do things like `powershell foo.ps1` or `powershell fooScript` without specifying `-File`.
+However, this change now requires that you explicitly specify `-c` or `-Command` when trying to do things like `powershell.exe Get-Command`.
+
+### Implement Unicode escape parsing [#3958](https://github.com/PowerShell/PowerShell/issues/3958)
+TODO
```

Comment:
I think you can just say:

\`u#### or \`u{####} is converted to the corresponding Unicode character.  To output a literal \`u, escape the backtick: ``u.